assets/docs/fragments/cli-installation.md

```diff
@@ -0,0 +1,21 @@
+## CLI Installation
+<details>
+<summary>MacOS</summary>
+
+You can install in MacOS via brew: `brew install asyncapi`.
+</details>
+
+<details>
+<summary>Linux</summary>
+
+You can install in Linux via `dpkg`, a debian package manager:
+
+1. `curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.deb` 
+2. `sudo dpkg -i asyncapi.deb`
+</details>
+
+<details>
+<summary>Other OS</summary>
+
+For further installation instructions for different operating systems, read the [`AsyncAPI CLI` readME file](https://github.com/asyncapi/cli#installation).
```

Comment:
I noticed already few people writing `readMe`, where is it coming from 😄 I've never seen this before 😄 why not just `read the AsyncAPI CLI docs`?

Reply:
`AsyncAPI CLI docs` is incorrect because the reader is already in the CLI docs. 

The readme is a clear and correct location. I think the spelling is wrong tho, so I will update it. (glad you pointed that out!)

Reply:
Ok confirming that [according to GH docs, the correct spelling should be: `README`](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes).